cmake/external_onnx.cmake

```diff
@@ -31,70 +30,31 @@ set(ONNX_GIT_REPO_URL https://github.com/onnx/onnx.git)
 set(ONNX_GIT_BRANCH rel-${ONNX_VERSION})
 
 add_definitions(-DONNX_BUILD_SHARED_LIBS=ON)
-add_definitions(-DONNX_ML=ON)
 
-ExternalProject_Add(
+if (WIN32)
+    string(REPLACE "/W3" "/W0" CMAKE_ORIGINAL_CXX_FLAGS "${CMAKE_ORIGINAL_CXX_FLAGS}")
+else()
+    string(REPLACE "-Werror" "" CMAKE_CXX_FLAGS "${CMAKE_ORIGINAL_CXX_FLAGS}")
+endif()
+
+FetchContent_Declare(
     ext_onnx
```

Comment:
Is it the project name or build library name? You have earlier changed the `ext_protobuf` dependency to `onnx`. This is just not clear to me.

Reply:
This is a name of FetchContent declaration AFAIK.
Library name will be known in line 53 when subdirectory will be added and onnx configuration step will finish.